### ISSUE TITLE
NS 3.0 HelloWorld won't run on iOS emulator with XCode 8.3.2

### ISSUE THREAD

**Issue by RobertGardner** [NativeScript/NativeScript#4160]
I ran into problems upgrading my current app to NativeScript 3.0 on iOS so I tried the HelloWorld tutorial and I am having the same problem.

I executed the following steps (on May 10, after the release of NS 3.0):

```
npm install -g nativescript
tns --version
  > 3.0.0
tns create HelloWorld --template nativescript-template-ng-tutorial
cd HelloWorld
tns run ios
```
Everything builds and the iOS emulator launches but then I get this error:

> Failed to install the requested application
> An application bundle was not found at the provided path.
> Provide a valid path to the desired application bundle.
> 
> while executing xcrun simctl install 2C26F15F-4194-4D29-AC48-24ECE8A58115 /Users/rgardner/Development/HelloWorld/platforms/ios/build/emulator/HelloWorld.app.
> 

I tried reinstalling platform ios and a few other suggestions I found, but nothing works.

I read in the RC notes that NS 3 doesn't support XCode 8.3 but I can't find that in the current docs, so I'm assuming it's fixed. I am running XCode 8.3.2.

**Comment by tsonevn** [comment]
Hi @RobertGardner,
Thank you for reporting this issue.
Indeed NativeScript 3.0 already supports Xcode 8.3 and you should be able to build your project while using it.
I tested your scenario on my side while using the above-given steps, however, I was unable to reproduce this case.
Perhaps this is related to an issue with generating the app file while building the project. It would help if you could send as the project while including the `platforms` folder.

**Comment by RobertGardner** [comment]
Thank you for looking at this! Just the knowledge that it should work helped. I did the following in the order listed and it is working now. Unfortunately, I'm not certain which step made the difference because I didn't think to do it slowly and meticulously. :-)

Completely uninstalled and reinstalled XCode (this did not fix it)
uninstalled and reinstalled nativescript (it's now version 3.0.1)
reinstalled xcodeproj and cocoapods
deleted and re-ran the HelloWorld tutorial steps (I saved the old version if you care to see it)
Works!

I suspect something was awry with my nativescript CLI update since I had some permissioning problems when I first did the upgrade, but who knows?

If you care, I can send you some further info, but I suspect that you'll be happy just closing this bug.

Thanks again!
